request/main_test.go

```diff
@@ -13,30 +13,36 @@ import (
 	"github.com/aws/aws-lambda-go/events"
 	"github.com/aws/aws-sdk-go-v2/aws/external"
 	"github.com/aws/aws-sdk-go-v2/service/acmpca"
+	"go/types"
+	"log"
 	mrand "math/rand"
+	"net/http"
+	"strings"
 	"testing"
 	"time"
 )
 
 const (
 	wrongResponseCode = "Request returned code: %d message: %s"
-)
-const (
-	acmpcaIssueCertificateRequest = `{
+
+	acmpcaListCertificateAuthoritiesRequest = `{"MaxResults": 100}`
+	acmpcaIssueCertificateRequest           = `{
```

Comment:
can you remove the extra blank spaces?

Reply:
That is a change made automatically by go formatter (go-fmt). Even if I remove the blank spaces, when trying to build Go will complain and request that I run go-fmt again and add the spaces back.